Configuration/PyReleaseValidation/scripts/runTheMatrix.py

```diff
@@ -65,15 +65,16 @@ def runSelected(opt):
                      4.22, #cosmic data
                      4.53, #run1 data + miniAOD
                      9.0, #Higgs200 charged taus
-                     1000, #data+prompt
-                     1001, #data+express
```

Comment:
I don't think this is a good idea. 
A lot of PCL AlCas are only exercised in `1001.0`, also the full battery of `@allForPrompt`is only exercised in `1000.0`

Reply:
Ok, I'll add them back for now. But I think it would be good to move those features to the recent prompt / express wf-s